.github/workflows/build_hip_basic.yml

```diff
@@ -0,0 +1,53 @@
+name: Build HIP-Basic
+
+on:
+  push:
+    branches: [ develop, release/** ]
+    paths:
+      - 'HIP-Basic/**'
+      - '.github/workflows/**'
+  pull_request:
+    branches: [ develop, release/** ]
+    paths:
+      - 'HIP-Basic/**'
+      - '.github/workflows/**'
+
+env:
+    ROCM_VERSION: 6.2
+    AMDGPU_INSTALLER_VERSION: 6.2.60200-1
+
+jobs:
+    build:
+        name: "Build HIP Examples"
+        runs-on: ubuntu-latest
+        container:
+            image: ubuntu:22.04
+        steps:
+          - uses: actions/checkout@v4
+
+          - name: Install dependencies
+            run: |
+                  apt-get update -qq &&
+                  apt-get install -y build-essential g++ glslang-tools \
+                    python3 python3-pip libglfw3-dev libvulkan-dev locales wget
+                  python3 -m pip install --upgrade pip
+                  python3 -m pip install cmake
+
+          - name: Install ROCm Dev
+            run: |
+                  export DEBIAN_FRONTEND=noninteractive
+                  wget https://repo.radeon.com/amdgpu-install/${{ env.ROCM_VERSION }}/ubuntu/jammy/amdgpu-install_${{ env.AMDGPU_INSTALLER_VERSION }}_all.deb
+                  apt-get -y install ./amdgpu-install_${{ env.AMDGPU_INSTALLER_VERSION }}_all.deb &&
+                  apt-get update -qq &&
+                  apt-get -y install rocm-dev rocm-llvm-dev
+                  echo "/opt/rocm/bin" >> $GITHUB_PATH
+                  echo "ROCM_PATH=/opt/rocm" >> $GITHUB_ENV
+                  echo "LD_LIBRARY_PATH=/opt/rocm/lib:${LD_LIBRARY_PATH}" >> $GITHUB_ENV
```

Comment:
This is wrong in many ways. It should not be needed, ROCm should be able to find its own libraries, /opt/rocm may not exist or it can point to the wrong place. If there is no existing LD_LIBRARY_PATH then you stick a spurious ":" on the end.